src/main/java/cn/nukkit/block/BlockWood.java

```diff
@@ -0,0 +1,71 @@
+package cn.nukkit.block;
+
+import cn.nukkit.item.Item;
+import cn.nukkit.item.ItemTool;
+import cn.nukkit.player.Player;
+import cn.nukkit.utils.BlockColor;
+import cn.nukkit.utils.Identifier;
+
+public class BlockWood extends BlockSolid {
+
+    public BlockWood(Identifier id) { super(id); }
+
+    @Override
+    public double getHardness() {
+        return 2;
+    }
+
+    @Override
+    public double getResistance() {
+        return 15;
+    }
+
+    @Override
+    public int getBurnChance() {
+        return 5;
+    }
+
+    @Override
+    public int getBurnAbility() {
+        return 20;
+    }
+
+    @Override
+    public int getToolType() {
+        return ItemTool.TYPE_AXE;
+    }
+
+    @Override
+    public boolean canBeActivated() {
+        return true;
```

Comment:
Missing a verification to check if the wood is already not stripped!

Reply:
Fixed at f27d01599d069e806745d22a28f1759a40e52d7a